lib/src/postgrest_error.dart

```diff
@@ -25,4 +25,10 @@ class PostgrestError {
         'hint': hint,
         'code': code,
       };
+
+  @override
+  String toString() {
+    return 'PostgrestError{message: $message, '
```

Comment:
I'd rather `PostgrestError(` over `{`

Reply:
its generated by the IDE not written by me :). 

The order of the variables are done by the IDE according to how they are defined in the class.

Reply:
@bdlukaa Changed `{` to `(` here! 
@k0shk0sh I hope you don't mind me doing this.